smart-send-logistics/smart-send-blocks-integration.php

```diff
@@ -0,0 +1,233 @@
+<?php
+use Automattic\WooCommerce\Blocks\Integrations\IntegrationInterface;
+
+/**
+ * Class for integrating with WooCommerce Blocks
+ */
+    class smart_send_Blocks_Integration implements IntegrationInterface {
```

Comment:
```suggestion
    class Smart_Send_Blocks_Integration implements IntegrationInterface {
```

Remember to fix this in all other implementations.

Reply:
fixed